src/core_write.cpp

```diff
@@ -157,6 +158,8 @@ void ScriptPubKeyToUniv(const CScript& scriptPubKey,
     int nRequired;
 
     out.pushKV("asm", ScriptToAsmStr(scriptPubKey));
+    out.pushKV("desc", InferDescriptor(scriptPubKey, DUMMY_SIGNING_PROVIDER)->ToString());
```

Comment:
Need to update help-debug of `gettxout` and also maybe `REST-interface.md` on `rest_getutxos`.

Do you want also to extend feature to `decodepsbt` ?

Reply:
> Do you want also to extend feature to decodepsbt ?

going to take a little more thinking, so for now "no" :)

fixed up other missing documentation to best of knowledge

Reply:
Oh I see `**ScriptToUniv**` is only used for PSBT decoding... so yeah I can do that real quick.

Reply:
For reference, `decodepsbt` *is* included in this pull (despite the docs not being updated)